### PR TITLE
Create ARecord Flywheel Service Provider Template

### PR DESCRIPTION
**Pull request by aftonpalmer**
Creates an ARecord Domain Connect Service Provider Template for Flywheel.

We would like to add a `logoUrl`. Could we have the logo shown below hosted at DomainConnect.org?

horz-blue

### CODE REVIEW

File: getflywheel.com.arecord.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+   "providerId":"getflywheel.com",
+   "providerName":"Flywheel",
+   "serviceId":"arecord",
+   "serviceName":"Delightfully Managed WordPress Hosting",
```
Comment (pawel-kow) [inline getflywheel.com.arecord.json:5]: Pls. reconsider shorter name. Such a long one may be cut by some UIs (like IONOS' one).

Comment (aftonpalmer) [reply getflywheel.com.arecord.json]: Update made and new commit pushed! 👍

**Comment by pawel-kow** [comment]
For `logoUrl` and hosting of logo on domainconnect.org I think this is possible as long as you list your service as Service Provider.
@rphilip can help to publish the entry.

**Comment by aftonpalmer** [comment]
> as long as you list your service as Service Provider.

Hoping to get clarification on this; by creating a Service Provider Template isn't that implying what we want is to be a Service Provider? Or is there an extra listing step that I'm maybe missing?

Sorry for the confusion! Just hope to get some final clarification, thank you.

**Comment by pawel-kow** [comment]
I mean listing here: https://www.domainconnect.org/dns-providers/

This is optional and not every provider wants that including contact email, that's why we don't do that by default.

**Comment by aftonpalmer** [comment]
> I mean listing here: https://www.domainconnect.org/dns-providers/
> 
> This is optional and not every provider wants that including contact email, that's why we don't do that by default.

Ah! Understood. Thank you for the clarification!  We are fine with being listed. Could we have our contact email set as `domainconnect@wpengine.com`?

**Comment by rphilip** [comment]
Hi @aftonpalmer once you have this template merged, please email admin@domainconnect.org with your logo, contact info and link to the template.  I'll get domainconnect.org updated to include Flywheel this week if you are able to send that info over.

Also I know there were previous talks with others between GoDaddy and WPEngine about Flywheel so if you are still interested in GoDaddy onboarding this template let me know.  DNS providers do not typically auto-onboard new templates.

**Comment by pawel-kow** [comment]
@rphilip it's a bit chicken and egg problem. They would like to host the logo on domainconnect.org and include in the template, so in fact it would be good to publish first to get the url.

**Comment by aftonpalmer** [comment]
> Hi @aftonpalmer once you have this template merged, please email [admin@domainconnect.org](mailto:admin@domainconnect.org) with your logo, contact info and link to the template. I'll get domainconnect.org updated to include Flywheel this week if you are able to send that info over.
> 
> Also I know there were previous talks with others between GoDaddy and WPEngine about Flywheel so if you are still interested in GoDaddy onboarding this template let me know. DNS providers do not typically auto-onboard new templates.

@rphilip Yes, absolutely, we would love for GoDaddy to onboard the template. Really appreciate the help there, thank you!

And to echo @pawel-kow, if our logo could be hosted via domainconnect.org, that would be much appreciated.

**Comment by rphilip** [comment]
@aftonpalmer  https://www.domainconnect.org/wp-content/uploads/2021/06/flywheel-logo.png.

**Comment by aftonpalmer** [comment]
Thank you @rphilip! New commit is ready with the new logoUrl added @pawel-kow

Let me know if you need anything else. We will definitely want this onboarded by GoDaddy, thank you very much for the assistance!

**Review by pawel-kow — APPROVED** [review]

**Comment by pawel-kow** [comment]
@aftonpalmer , it's merged now. For IONOS we are in contact with Carl about onboarding.

**Pull request merged**